Common/src/CConfig.cpp

```diff
@@ -1385,6 +1387,13 @@ void CConfig::SetConfig_Options() {
   flame_init[6] = 0.5e-3; flame_init[7] = 1.0;
   addDoubleArrayOption("FLAME_INIT", 8,flame_init);
 
+  /*!\brief SPARK_INIT \n DESCRIPTION: spark initialization using the flamelet model \ingroup Config*/
+  for (auto iSpark=0u; iSpark<6; ++iSpark) spark_init[iSpark]=0;
+  addDoubleArrayOption("SPARK_INIT", 6, spark_init);
```

Comment:
Can you use std::array? I forget if it's compatible.

Reply:
Do you mean defining `spark_init` as a `std::array<su2double, 6>` ? Unfortunately no, as then I'd have to define separate getter functions for retrieving each kind of flame initialization method quantities.

Reply:
you can still return array.data()

Reply:
> Can you use std::array? I forget if it's compatible.

It is compatible if you use `array.begin()` as argument in the `AddDoubleArrayOption` function.